library/core/src/num/nonzero.rs

```diff
@@ -849,7 +849,7 @@ macro_rules! nonzero_integer {
 
 macro_rules! nonzero_integer_signedness_dependent_impls {
     // Impls for unsigned nonzero types only.
-    ($Ty:ident unsigned $Int:ty) => {
+    (unsigned $Int:ty) => {
```

Comment:
Are these changes just driveby cleanup?

Reply:
Yup; leftover unused code from https://github.com/rust-lang/rust/pull/126904 (which came between my original PR and now).